### DISCUSSION TITLE
[Unstable] Interacting with fire extinguisher brackets and weapon holders no longer automatically picks up the item

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14864] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

.

### Reproduction steps

1. interact with fire extinguisher brackets

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.17.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Are you sure it's a bug and not a satisfied feature request (xd)? https://github.com/FakeFishGames/Barotrauma/discussions/14425

**Comment by lIllIIlI** [comment]
This is not a bug. Accidentally picking up fire extinguishers when trying to open doors and what not was annoying. If the extra click to pick up a fire extinguisher to extinguish an ongoing fire is too much, you were too late to begin with.

**Reply by SomeRandomNoobKekeke** [reply]
Whatever it was, it's fixed https://github.com/FakeFishGames/Barotrauma/issues/14867
![BaroDev (wide)](https://github.com/user-attachments/assets/bd9cfa03-b935-4b71-a347-26ce1fc20c0c)

**Reply by OverwatchVoice** [reply]
To be fair, regalis or any other developers never acknowledged the [feature request](https://github.com/FakeFishGames/Barotrauma/discussions/14425), so it could've been a bug, but one that some people requested.

**Comment by Jasontti** [comment]
Tested a fix for this.
Extinguishers, etc are now properly picked up when AutoInteractWithContained is true on item definition

Do note, that this fix is for this feature only. Should they be picked up automatically should be made in to a new discussion.

Closing discussion as resolved.
